lib/common/cmdline.c

```diff
@@ -89,15 +89,15 @@ pcmk__build_arg_context(pcmk__common_args_t *common_args, const char *fmts,
         GOptionEntry output_entries[3] = {
             { "output-as", 0, 0, G_OPTION_ARG_STRING, &(common_args->output_ty),
               NULL,
-              "FORMAT" },
+              N_("FORMAT") },
             { "output-to", 0, 0, G_OPTION_ARG_STRING, &(common_args->output_dest),
-              "Specify file name for output (or \"-\" for stdout)", "DEST" },
+             N_( "Specify file name for output (or \"-\" for stdout)"),N_("DEST") },
```

Comment:
Rather a nitpicking than a real problem, but could we keep the whitespace after the `,`  in between for the sake of readability? This really is a big struct with a long list of variables :-)

Reply:
ok,I will fix it:-)